README.md
# BabyLogger
Native SailfishOS application for monitoring/logging activity of a human baby


CHANGELOG
Changelog:
==========

0.4
-----
- Added meal support
- Delete support
- Complete UI makeover

0.4.1
-----
- Many bugfixes (Fixed #11, Fixed #9, Fixed #6, Fixed #1)
- Added last meal timer issue #12

0.4.2
-----
- Fixed #7 CoverPage layout
- Fixed #13 Listview section partitioning
- Code cleanup

0.4.3
-----
- Fixed #15 (about page) + moved listview delegate to diff file
- Fixed some UI bugs in about page
- Merged pull-request from Rikujolla (Finnish translation)

0.4.4
-----
- Add settings page / component Fixes #16
- Counter timers now go idle when app is inactive Fixes #22
- Added signals for meal/action add/update/delete events
- Added 'clear all logged data' option (button in settings page) Fixes #21
- Fresh/Cleared install now shows information text and clean counters Fixes #19
- many small bugfixes eg: Fixed #24
- Set release version to 0.4.4

0.4.6
-----
- Improved log meal dialog: Fixes #38, Fixes #5
- Updated application icon
- Added coveractions for toggleSleep and meal; Fixes #36
- Fixes #2 : per day statistics (experimental)
- Bugfixes


translations/baby_app.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1">
<context>
    <name>CoverPage</name>
    <message>
        <source>Baby logger 0.1</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>FirstPage</name>
    <message>
        <source>Baby Logger</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Show logs</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>sleeping...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>awake...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Stop sleep</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Start sleep</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>SecondPage</name>
    <message>
        <source>Baby logs</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Set time</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Set date</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>


translations/baby-logger-de.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1">
<context>
    <name>CoverPage</name>
    <message>
        <source>My Cover</source>
        <translation type="vanished">Mein Cover</translation>
    </message>
    <message>
        <source>Baby logger 0.1</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>FirstPage</name>
    <message>
        <source>Show Page 2</source>
        <translation type="vanished">Zur Seite 2</translation>
    </message>
    <message>
        <source>UI Template</source>
        <translation type="vanished">UI-Vorlage</translation>
    </message>
    <message>
        <source>Hello Sailors</source>
        <translation type="vanished">Hallo Matrosen</translation>
    </message>
    <message>
        <source>Baby Logger</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Show logs</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>sleeping...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>awake...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Stop sleep</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Start sleep</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>SecondPage</name>
    <message>
        <source>Nested Page</source>
        <translation type="vanished">Unterseite</translation>
    </message>
    <message>
        <source>Item</source>
        <translation type="vanished">Element</translation>
    </message>
    <message>
        <source>Baby logs</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Set time</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Set date</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>


translations/harbour-babylogger-de.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1">
<context>
    <name>BabyModel</name>
    <message>
        <source>Breast milk</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Formula</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Pureed food</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>Counter</name>
    <message>
        <source>sleeping...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>awake...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>since last meal</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>CoverPage</name>
    <message>
        <source>My Cover</source>
        <translation type="vanished">Mein Cover</translation>
    </message>
</context>
<context>
    <name>FirstPage</name>
    <message>
        <source>Show Page 2</source>
        <translation type="vanished">Zur Seite 2</translation>
    </message>
    <message>
        <source>UI Template</source>
        <translation type="vanished">UI-Vorlage</translation>
    </message>
    <message>
        <source>Hello Sailors</source>
        <translation type="vanished">Hallo Matrosen</translation>
    </message>
    <message>
        <source>Show logs</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Baby Logger</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>sleeping...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>awake...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Stop sleep</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Start sleep</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>LogMealDialog</name>
    <message>
        <source>Type</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Quantity</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Common choices:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Breast milk</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Formula 60ml</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Formula 120ml</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Add</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>Overview</name>
    <message>
        <source>Show logs</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Stop sleep</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Start sleep</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Baby Logger</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Log Meal</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Actions Menu</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>SecondPage</name>
    <message>
        <source>Nested Page</source>
        <translation type="vanished">Unterseite</translation>
    </message>
    <message>
        <source>Item</source>
        <translation type="vanished">Element</translation>
    </message>
    <message>
        <source>Set time</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Delete</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Log Meal</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Stop sleep</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Start sleep</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Actions Menu</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Updating time</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Deleting entry</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Sleeping</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Awake</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>harbour-babylogger</name>
    <message>
        <source>Baby Logger</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>0.4.1</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>


translations/harbour-babylogger-fi.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1">
<context>
    <name>AboutPage</name>
    <message>
        <source>About</source>
        <translation>Tietoja</translation>
    </message>
    <message>
        <source>Native SailfishOS application for monitoring/logging activity of a human baby.</source>
        <translation>Vauvan seurantaan tarkoitettu SailfishOS-sovellus</translation>
    </message>
    <message>
        <source>Developer</source>
        <translation>Kehittäjä</translation>
    </message>
    <message>
        <source>Goldfrapper &lt;goldfrapper@gmail.com&gt;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>GitHub Repository</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>https://github.com/goldfrapper/BabyLogger/</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Icon</source>
        <translation>Ikoni</translation>
    </message>
    <message>
        <source>Icon based on the artwork of Yazmin Alanis (http://thenounproject.com/yalanis/)</source>
        <translation>Ikoni perustuu Yazmin Alanisin kuvamateriaaliin (http://thenounproject.com/yalanis/)</translation>
    </message>
    <message>
        <source>Translations</source>
        <translation>Käännökset</translation>
    </message>
    <message>
        <source>Licence</source>
        <translation>Lisenssi</translation>
    </message>
</context>
<context>
    <name>BabyModel</name>
    <message>
        <source>Breast milk</source>
        <translation>Äidinmaito</translation>
    </message>
    <message>
        <source>Formula</source>
        <translation>Korvike</translation>
    </message>
    <message>
        <source>Pureed food</source>
        <translation>Sose</translation>
    </message>
    <message>
        <source>Sleep stopped</source>
        <translation>Heräsi</translation>
    </message>
    <message>
        <source>Sleep started</source>
        <translation>Nukahti</translation>
    </message>
    <message>
        <source>Meal</source>
        <translation>Ateria</translation>
    </message>
    <message>
        <source>Update failed</source>
        <translation>Päivitys epäonnistui</translation>
    </message>
    <message>
        <source>The given time must not be in the future</source>
        <translation>Syötetty aika ei voi olla tulevaisuudessa</translation>
    </message>
    <message>
        <source>You cannot update beyond the boundaries of adjoining sleep logs</source>
        <translation>Aika-arvoa ei voi päivittää edellisen tai seuraavan unijakson päälle</translation>
    </message>
</context>
<context>
    <name>Counter</name>
    <message>
        <source>sleeping...</source>
        <translation>nukkuu...</translation>
    </message>
    <message>
        <source>awake...</source>
        <translation>hereillä...</translation>
    </message>
    <message>
        <source>since last meal</source>
        <translation>ateriasta</translation>
    </message>
</context>
<context>
    <name>FirstPage</name>
    <message>
        <source>Stop sleep</source>
        <translation>Herääminen</translation>
    </message>
    <message>
        <source>Start sleep</source>
        <translation>Nukahtaminen</translation>
    </message>
    <message>
        <source>About</source>
        <translation>Tietoja</translation>
    </message>
    <message>
        <source>Log Meal</source>
        <translation>Kirjaa ateria</translation>
    </message>
    <message>
        <source>Actions Menu</source>
        <translation type="vanished">Toiminnot</translation>
    </message>
    <message>
        <source>Settings</source>
        <translation>Asetukset</translation>
    </message>
    <message>
        <source>You have not yet logged anything. Start by selecting &apos;Start sleep&apos; or &apos;Log meal&apos; from the pull-down menu.</source>
        <translation>Et ole vielä kirjannut mitään. Aloita valitsemalla alasvetovalikosta &apos;Nukahtaminen&apos; tai &apos;Kirjaa ateria&apos;.</translation>
    </message>
    <message>
        <source>total sleep</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>LogEntryItem</name>
    <message>
        <source>Sleeping</source>
        <translation>Nukkunut</translation>
    </message>
    <message>
        <source>Awake</source>
        <translation>Hereillä</translation>
    </message>
    <message>
        <source>Set time</source>
        <translation>Aseta aika</translation>
    </message>
    <message>
        <source>Updating time</source>
        <translation>Päivitetään aika</translation>
    </message>
    <message>
        <source>Delete</source>
        <translation>Poista</translation>
    </message>
    <message>
        <source>Deleting entry</source>
        <translation>Poistetaan</translation>
    </message>
</context>
<context>
    <name>LogMealDialog</name>
    <message>
        <source>Type</source>
        <translation type="vanished">Tyyppi</translation>
    </message>
    <message>
        <source>Quantity</source>
        <translation>Määrä</translation>
    </message>
    <message>
        <source>Common choices:</source>
        <translation>Pikavalinnat:</translation>
    </message>
    <message>
        <source>Add</source>
        <translation>Lisää</translation>
    </message>
    <message>
        <source>Time since start of meal:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>This amount is automaticly substracted from the current time</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Meal Type</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Select the type of food given. Currently supported types are breastmilk, formula and pureed food.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>NotificationDialog</name>
    <message>
        <source>Back</source>
        <translation>Takaisin</translation>
    </message>
</context>
<context>
    <name>SettingsPage</name>
    <message>
        <source>Settings</source>
        <translation>Asetukset</translation>
    </message>
    <message>
        <source>Clear all logs</source>
        <translation>Poista kaikki kirjaukset</translation>
    </message>
    <message>
        <source>Max. Awake Time</source>
        <translation>Maksimi hereilläoloaika</translation>
    </message>
    <message>
        <source>Max. &apos;Between meals&apos; Time</source>
        <translation>Maksimi ateriaväli</translation>
    </message>
    <message>
        <source>Alerts and Timers</source>
        <translation>Hälytykset ja ajastimet</translation>
    </message>
    <message>
        <source>Application Reset</source>
        <translation>Sovelluksen tyhjennys</translation>
    </message>
    <message>
        <source>This value controls when the application should warn you that the baby has been awake for too long</source>
        <translation>Tämän aika-arvoasetuksen perusteella sovellus varoittaa, milloin vauva on ollut hereillä liian pitkään.</translation>
    </message>
    <message>
        <source>This is the maximum time that should be allowed between meals. This greatly depends on the age of the baby</source>
        <translation>Tämä on ateriavälin maksimiaika-asetus. Tämä riippuu voimakkaasti vauvan iästä.</translation>
    </message>
    <message>
        <source>Reset Application</source>
        <translation>Tyhjennä sovellus</translation>
    </message>
    <message>
        <source>This will remove all saved data. Are you shure this is what you want? There is no easy way to undo this</source>
        <translation>Tämä poistaa kaikki tallentamasi tiedot. Oletko varma, että haluat tätä? Tietojen palauttamiseen ei ole helppoa tapaa.</translation>
    </message>
</context>
<context>
    <name>harbour-babylogger</name>
    <message>
        <source>Baby Logger</source>
        <translation>Vauvan seuranta</translation>
    </message>
    <message>
        <source>0.4.6</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>


translations/harbour-babylogger.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1">
<context>
    <name>AboutPage</name>
    <message>
        <location filename="../qml/pages/AboutPage.qml" line="19"/>
        <source>About</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/AboutPage.qml" line="39"/>
        <source>Native SailfishOS application for monitoring/logging activity of a human baby.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/AboutPage.qml" line="48"/>
        <source>Developer</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/AboutPage.qml" line="49"/>
        <source>Goldfrapper &lt;goldfrapper@gmail.com&gt;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/AboutPage.qml" line="54"/>
        <source>GitHub Repository</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/AboutPage.qml" line="55"/>
        <source>https://github.com/goldfrapper/BabyLogger/</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/AboutPage.qml" line="60"/>
        <source>Icon</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/AboutPage.qml" line="61"/>
        <source>Icon based on the artwork of Yazmin Alanis (http://thenounproject.com/yalanis/)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/AboutPage.qml" line="67"/>
        <source>Translations</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/AboutPage.qml" line="74"/>
        <source>Licence</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>BabyModel</name>
    <message>
        <location filename="../qml/pages/BabyModel.qml" line="11"/>
        <source>Breast milk</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/BabyModel.qml" line="11"/>
        <source>Formula</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/BabyModel.qml" line="11"/>
        <source>Pureed food</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/BabyModel.qml" line="13"/>
        <source>Sleep stopped</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/BabyModel.qml" line="14"/>
        <source>Sleep started</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/BabyModel.qml" line="15"/>
        <source>Meal</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/BabyModel.qml" line="514"/>
        <location filename="../qml/pages/BabyModel.qml" line="527"/>
        <source>Update failed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/BabyModel.qml" line="515"/>
        <source>The given time must not be in the future</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/BabyModel.qml" line="528"/>
        <source>You cannot update beyond the boundaries of adjoining sleep logs</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>Counter</name>
    <message>
        <location filename="../qml/pages/Counter.qml" line="87"/>
        <source>since last meal</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/Counter.qml" line="111"/>
        <source>sleeping...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/Counter.qml" line="111"/>
        <source>awake...</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>FirstPage</name>
    <message>
        <location filename="../qml/pages/FirstPage.qml" line="47"/>
        <source>You have not yet logged anything. Start by selecting &apos;Start sleep&apos; or &apos;Log meal&apos; from the pull-down menu.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/FirstPage.qml" line="106"/>
        <source>total sleep</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/FirstPage.qml" line="128"/>
        <source>About</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/FirstPage.qml" line="134"/>
        <source>Settings</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/FirstPage.qml" line="140"/>
        <source>Log Meal</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/FirstPage.qml" line="165"/>
        <source>Stop sleep</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/FirstPage.qml" line="165"/>
        <source>Start sleep</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>LogEntryItem</name>
    <message>
        <location filename="../qml/pages/LogEntryItem.qml" line="69"/>
        <source>Sleeping</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/LogEntryItem.qml" line="69"/>
        <source>Awake</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/LogEntryItem.qml" line="110"/>
        <source>Set time</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/LogEntryItem.qml" line="126"/>
        <source>Updating time</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/LogEntryItem.qml" line="155"/>
        <source>Delete</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/LogEntryItem.qml" line="158"/>
        <source>Deleting entry</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>LogMealDialog</name>
    <message>
        <location filename="../qml/pages/LogMealDialog.qml" line="19"/>
        <source>Add</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/LogMealDialog.qml" line="26"/>
        <source>Meal Type</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/LogMealDialog.qml" line="29"/>
        <source>Select the type of food given. Currently supported types are breastmilk, formula and pureed food.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/LogMealDialog.qml" line="48"/>
        <source>Quantity</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/LogMealDialog.qml" line="69"/>
        <source>Common choices:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/LogMealDialog.qml" line="95"/>
        <source>Time since start of meal:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/LogMealDialog.qml" line="96"/>
        <source>This amount is automaticly substracted from the current time</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>NotificationDialog</name>
    <message>
        <location filename="../qml/pages/NotificationDialog.qml" line="16"/>
        <source>Back</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>SettingsPage</name>
    <message>
        <location filename="../qml/pages/SettingsPage.qml" line="21"/>
        <source>Settings</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/SettingsPage.qml" line="25"/>
        <source>Alerts and Timers</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/SettingsPage.qml" line="46"/>
        <source>This value controls when the application should warn you that the baby has been awake for too long</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/SettingsPage.qml" line="67"/>
        <source>This is the maximum time that should be allowed between meals. This greatly depends on the age of the baby</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/SettingsPage.qml" line="73"/>
        <source>Application Reset</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/SettingsPage.qml" line="78"/>
        <source>Clear all logs</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/SettingsPage.qml" line="91"/>
        <source>Reset Application</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/SettingsPage.qml" line="92"/>
        <source>This will remove all saved data. Are you shure this is what you want? There is no easy way to undo this</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/SettingsPage.qml" line="30"/>
        <source>Max. Awake Time</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/pages/SettingsPage.qml" line="51"/>
        <source>Max. &apos;Between meals&apos; Time</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>harbour-babylogger</name>
    <message>
        <location filename="../qml/harbour-babylogger.qml" line="10"/>
        <source>Baby Logger</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qml/harbour-babylogger.qml" line="11"/>
        <source>0.4.6</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
